corehq/messaging/fcm_util.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import os
+
+from django.conf import settings
+from firebase_admin import messaging, credentials, initialize_app
+
+logger = logging.getLogger(__name__)
+
+MAX_DEVICES_ALLOWED_MULTICAST = 500
+HQ_FCM_UTIL = None
+
+
+def is_fcm_available():
+    if not settings.FCM_CREDS_PATH:
+        return False
+    return os.path.isfile(settings.FCM_CREDS_PATH)
+
+
+class FCMUtilException(Exception):
+    pass
+
+
+class DevicesLimitExceeded(FCMUtilException):
+    def __init__(self):
+        super().__init__(f"Max devices allowed is {MAX_DEVICES_ALLOWED_MULTICAST}! Please execute in batches.")
+
+
+class FCMUtil:
+    def __init__(self, app_name, creds_path=settings.FCM_CREDS_PATH):
```

Comment:
do we need to pass a different creds_path? I assume the only we are supporting is `settings.FCM_CREDS_PATH`

Reply:
I designed this in a way so that the consumer of this class can create multiple apps with different creds default app being `HQ_FCM_UTIL`.
However we have one Commcare App (can be linked to only project) so practically I don't multiple apps/creds getting created in HQ.
Your thoughts ?

Reply:
@mkangia
Resolved https://github.com/dimagi/commcare-hq/commit/40fc5b619e4ac6202030448ae3661c7a5e52b327

Reply:
>  However we have one Commcare App (can be linked to only project) so practically I don't see multiple apps/creds getting created in HQ so maybe support only settings.FCM_CREDS_PATH.

Ya. Lets stick with YAGNI principle and keep it simple for now i.e support only one setting in localsettings.